src/server.ts

```diff
@@ -15,7 +15,7 @@ const initializeServer = () => {
   server.get("/projects/:projectId", project);
 
   // ticket
-  server.put("/tickets/:ticketId", editTicket);
+  server.put("/tickets/:ticketId", ticket.editTicket);
```

Comment:
I guess you forgot import here

Reply:
Whoops, I'll fix that up with the next PR coming up